common-template/src/main/java/com/redhat/cloud/notifications/qute/templates/mapping/Rhel.java

```diff
@@ -0,0 +1,74 @@
+package com.redhat.cloud.notifications.qute.templates.mapping;
+
+import com.redhat.cloud.notifications.qute.templates.TemplateDefinition;
+import java.util.Map;
+
+import static com.redhat.cloud.notifications.qute.templates.IntegrationType.DRAWER;
+import static java.util.Map.entry;
+
+public class Rhel {
+    static final String BUNDLE_NAME = "rhel";
+
+    static final String ADVISOR_APP_NAME = "advisor";
+    static final String ADVISOR_FOLDER_NAME = "Advisor";
+
+    static final String COMPLIANCE_APP_NAME = "compliance";
+    static final String COMPLIANCE_FOLDER_NAME = "Compliance";
+
+    static final String DRIFT_APP_NAME = "drift";
+    static final String DRIFT_FOLDER_NAME = "Drift";
+
+    static final String EDGE_APP_NAME = "edge-management";
+    static final String EDGE_FOLDER_NAME = "EdgeManagement";
+
+    static final String INVENTORY_APP_NAME = "inventory";
+    static final String INVENTORY_FOLDER_NAME = "Inventory";
+
+    static final String MALWARE_APP_NAME = "malware-detection";
+    static final String MALWARE_FOLDER_NAME = "MalwareDetection";
+
+    static final String PATCH_APP_NAME = "patch";
+    static final String PATCH_FOLDER_NAME = "Patch";
+
+    static final String POLICY_APP_NAME = "policies";
+    static final String POLICY_FOLDER_NAME = "Policies";
+
+    static final String VULNERABILITY_APP_NAME = "vulnerability";
+    static final String VULNERABILITY_FOLDER_NAME = "Vulnerability";
+
+    public static final Map<TemplateDefinition, String> templatesMap = Map.ofEntries(
+        // Advisor
+        entry(new TemplateDefinition(DRAWER, BUNDLE_NAME, ADVISOR_APP_NAME, "deactivated-recommendation"), ADVISOR_FOLDER_NAME + "/deactivatedRecommendationBody.md"),
+        entry(new TemplateDefinition(DRAWER, BUNDLE_NAME, ADVISOR_APP_NAME, "new-recommendation"), ADVISOR_FOLDER_NAME + "/newRecommendationBody.md"),
+        entry(new TemplateDefinition(DRAWER, BUNDLE_NAME, ADVISOR_APP_NAME, "resolved-recommendation"), ADVISOR_FOLDER_NAME + "/resolvedRecommendationBody.md"),
+
+        // Compliance
+        entry(new TemplateDefinition(DRAWER, BUNDLE_NAME, COMPLIANCE_APP_NAME, "compliance-below-threshold"), COMPLIANCE_FOLDER_NAME + "/belowThresholdBody.md"),
+        entry(new TemplateDefinition(DRAWER, BUNDLE_NAME, COMPLIANCE_APP_NAME, "report-upload-failed"), COMPLIANCE_FOLDER_NAME + "/reportUploadFailedBody.md"),
+
+        // Drift
+        entry(new TemplateDefinition(DRAWER, BUNDLE_NAME, DRIFT_APP_NAME, "drift-baseline-detected"), DRIFT_FOLDER_NAME + "/newBaselineDriftBody.md"),
```

Comment:
```suggestion
```

Reply:
Ideally, adding a `TemplateDefinition` that references something that doesn't exist shouldn't be possible, and covered with a unit test.